client/src/ui/organisms/footer/index.tsx

```diff
@@ -225,7 +225,7 @@ export function Footer() {
             <br />
             Portions of this content are ©1998–{new Date().getFullYear()} by
             individual mozilla.org contributors. Content available under{" "}
-            <a href="/docs/MDN/About#Copyrights_and_licenses">
+            <a href="/docs/MDN/Writing_guidelines/Attrib_copyright_license">
```

Comment:
Could you please also add the locale to the url, like in lines 135 and 140? Thanks! 🙏

Reply:
Okay, I am working on it.

Reply:
Do I need to change all content present on https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Attrib_copyright_license in different languages and then add the locale to the URL that basically helps to change language like in 135 and 140? Also, I check that page by changing en-US to ru for Russian or ko for Korean in the URL the page is not present in those languages but other pages are available in those languages.

Reply:
You can check this as well
![Screenshot 2022-11-15 221933](https://user-images.githubusercontent.com/109434545/201978451-eb9d85af-2646-4649-af4d-58e4d3e6e59c.png)

Reply:
So how to achieve adding locale to the URL do I use <a href={`/${locale}//docs/MDN/Writing_guidelines/Attrib_copyright_license`} only and remove the earlier link from there?